### PR TITLE
fix meta in mls

### PR DESCRIPTION
**Pull request by polinaeterna**
`monolingual` value of `m ultilinguality` param in yaml meta was changed to `multilingual` :)

### CODE REVIEW

**Review by patrickvonplaten — APPROVED** [review]

**Review by lhoestq — APPROVED** [review]
good catch !

**Comment by patrickvonplaten** [comment]
Feel free to merge @polinaeterna as soon as you got an approval from either @lhoestq , @albertvillanova or @mariosasko

**Pull request merged**
